content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -110,6 +110,13 @@ using the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-co
 These settings let you configure the maximum size for each log file and the maximum number of
 files allowed for each container respectively.
 
+In order to perform an efficient log rotation in clusters where the volume of the logs generated by
+the workload is large, kubelet also provides a mechanism to tune how the logs are rotated in
+terms of how many concurrent log rotations can be performed and the interval at which the logs are
+monitored and rotated as required. These attributes can be configured by setting `containerLogMaxWorkers`
```

Comment:
I'd add that these are kubelet configuration flags.

Look above at rotate and follow the same pattern.

Reply:
> You can configure two kubelet [configuration settings](https://github.com/kubernetes/website/blob/0ca3f289c362ebc1728df80babe4367800a7f8f2/docs/reference/config-api/kubelet-config.v1beta1), containerLogMaxSize and containerLogMaxFiles, using the [kubelet configuration file](https://github.com/kubernetes/website/blob/0ca3f289c362ebc1728df80babe4367800a7f8f2/docs/tasks/administer-cluster/kubelet-config-file). These settings let you configure the maximum size for each log file and the maximum number of files allowed for each container respectively.

Maybe you can follow something similar for these two parameters.

Reply:
Ack

Reply:
@kannon92  should I add a note to indicate this is only added in 1.30 for now ?